packages/web-forms/src/components/form-elements/input/InputControl.vue

```diff
@@ -8,13 +8,17 @@ import InputDecimal from '@/components/form-elements/input/InputDecimal.vue';
 import InputInt from '@/components/form-elements/input/InputInt.vue';
 import InputNumbersAppearance from '@/components/form-elements/input/InputNumbersAppearance.vue';
 import InputText from '@/components/form-elements/input/InputText.vue';
+import { IS_FORM_EDIT_MODE } from '@/lib/constants/injection-keys.ts';
 import type { AnyInputNode } from '@getodk/xforms-engine';
+import { inject } from 'vue';
 
 interface InputControlProps {
 	readonly node: AnyInputNode;
 }
 
 defineProps<InputControlProps>();
+
+const isFormEditMode = inject(IS_FORM_EDIT_MODE, false);
```

Comment:
Regarding the suggestion in `InputGeopointWithMap`, whatever you go with there, make this consistent.